corehq/apps/userreports/reports/filters.py

```diff
@@ -52,7 +52,9 @@ def __init__(self, filter, value):
     def to_sql_filter(self):
         if self.value is None:
             return ""
-        return "{0} {1} :operand".format(self.filter.field, self.value['operator'])
+        ret = BasicFilter(self.filter.field, 'operand')
+        ret.operator = self.value['operator']
```

Comment:
Seems totally reasonable to add `operator` to the `__init__` method there as a kwarg that defaults to `None`.

Once the change has been made you just need to bump there version and then run the command:
```
python setup.py sdist upload
```